Source/ThirdParty/SDL/src/core/android/SDL_android.c

```diff
@@ -718,20 +718,31 @@ JNIEXPORT void JNICALL SDL_JAVA_CONTROLLER_INTERFACE(nativeSetupJNI)(JNIEnv *env
 /* SDL main function prototype */
 typedef int (*SDL_main_func)(int argc, char *argv[]);
 
+static SDL_main_func SDL_external_main = 0;
+
+/* Custom SDL_main_callback */
+JNIEXPORT void SetExternalSdlMain(SDL_main_func external_main)
```

Comment:
WTF is this patch?.. I mean, maybe it's needed, but it looks so cursed. Imagine migrating it to SDL3

Reply:
Diff does not do this code justice tbh. With a better diff editor it is clearly visible that changes are minimal and it looks ugly only because of indentation. We do want to not have it for sure, but imo we can figure it out a bit later.